### ISSUE TITLE
P1 Standing Fizzler X1 top tiles are being invisible in a "Mysterious Way"

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#2457]
Please, insert the x files theme song here: 
![Half Life 3 Confirmed](https://user-images.githubusercontent.com/12244376/35289551-6f07f720-0046-11e8-9501-8da7ad24f39d.jpg)

**Comment by LautaroL20** [comment]
Now fixed.

**Comment by ncrecc** [comment]
If it's fixed, shouldn't it be closed?

**Comment by LautaroL20** [comment]
It will be closed automatically once commit "a707e31" is merged into the "master" branch, which will be when Pre-release 29 is out. Meantwhile, people who sees fixed issues will be able to notice which bugs are being fixed.
